### ISSUE TITLE
Add missing title input on gutenberg dashboard

### ISSUE THREAD

**Issue by maxime-pasquier** [Automattic/syntaxhighlighter#120]
Hello everyone,

I am asking if there is a reason the input for title is missing on the gutenberg block on dashboard ?
![Capture du 2020-02-12 10-02-05](https://user-images.githubusercontent.com/6303296/74319292-faf80e00-4d7e-11ea-8764-34f03da61845.png)

It will be so great, don't you thinks ?

**Comment by maxime-pasquier** [comment]
And one more yes/no input to display the title (or not) !

Thanks!

**Comment by GTOL** [comment]
Please add this function!

**Comment by yscik** [comment]
To clarify, this is for an option to show/hide the **Title** option — specified in the plugin settings — above the block, right?

And also to make this title editable separately for each block?

**Comment by maxime-pasquier** [comment]
Yes, for example, I do not add a default title (on settings) to hide the title on my blocks (for the moment) :
https://cameliaweb.fr/blog/jeux-video-gaming/installer-wine-sur-ubuntu-pour-les-versions-lts

But I would to set a different title for each block, with a default title, and sometimes hide the title on one block.

So basically, an input title name for each block, and a yes/no for displaying the title or not.

Thanks !
